src/prefect/workers/process.py

```diff
@@ -249,279 +218,28 @@ async def run(
+        process = await self._runner.execute_flow_run(
+            flow_run_id=flow_run.id,
+            command=configuration.command,
+            cwd=configuration.working_dir,
+            env=configuration.env,
+            stream_output=configuration.stream_output,
+            task_status=task_status,
         )
-        with working_dir_ctx as working_dir:
-            flow_run_logger.debug(
-                f"Process running command: {command} in {working_dir}"
-            )
-            process = await run_process(
-                command.split(" "),
-                stream_output=configuration.stream_output,
-                task_status=task_status,
-                task_status_handler=_infrastructure_pid_from_process,
-                cwd=working_dir,
-                env=configuration.env,
-                **kwargs,
-            )
-
-        # Use the pid for display if no name was given
-        display_name = f" {process.pid}"
-
-        if process.returncode:
-            help_message = None
-            if process.returncode == -9:
-                help_message = (
-                    "This indicates that the process exited due to a SIGKILL signal. "
-                    "Typically, this is either caused by manual cancellation or "
-                    "high memory usage causing the operating system to "
-                    "terminate the process."
-                )
-            if process.returncode == -15:
-                help_message = (
-                    "This indicates that the process exited due to a SIGTERM signal. "
-                    "Typically, this is caused by manual cancellation."
-                )
-            elif process.returncode == 247:
-                help_message = (
-                    "This indicates that the process was terminated due to high "
-                    "memory usage."
-                )
-            elif (
-                sys.platform == "win32" and process.returncode == STATUS_CONTROL_C_EXIT
-            ):
-                help_message = (
-                    "Process was terminated due to a Ctrl+C or Ctrl+Break signal. "
-                    "Typically, this is caused by manual cancellation."
-                )
-
-            flow_run_logger.error(
-                f"Process{display_name} exited with status code: {process.returncode}"
-                + (f"; {help_message}" if help_message else "")
-            )
-        else:
-            flow_run_logger.info(f"Process{display_name} exited cleanly.")
+
+        if process is None or process.returncode is None:
+            raise RuntimeError("Failed to start flow run process.")
 
         return ProcessWorkerResult(
             status_code=process.returncode, identifier=str(process.pid)
         )
 
-    async def kill_process(
-        self,
-        infrastructure_pid: str,
-        grace_seconds: int = 30,
-    ) -> None:
-        hostname, pid = _parse_infrastructure_pid(infrastructure_pid)
```

Comment:
are these used at this point?

https://github.com/prefecthq/prefect/blob/use-runner-in-process-worker/src/prefect/workers/process.py#L53-L60

Reply:
Nope, I removed them